CMakeLists.txt

```diff
@@ -58,6 +58,8 @@ else()
 endif()
 
 option(WITH_CTEST "Enable ctest integration of tests" ON)
+option(DISABLE_LTO "Disable LTO for faster linking" OFF)
```

Comment:
Could this be changed to `ENABLE_LTO`?

Ideally setting a boolean to `ON` would enable something, not disable it.